schema/config-schema.json

```diff
@@ -97,6 +118,10 @@
                 "user": {
                     "id": "https://opencontainers.org/schema/bundle/process/user",
                     "type": "object",
+                    "required": [
+			"uid",
+			"gid"
```

Comment:
These are [only required on Linux/Solaris][1], and there's no way to enforce that constraint in JSON Schema without some serious `oneOf` hoop-jumping.

[1]: https://github.com/opencontainers/runtime-spec/blob/ced3365b4d5856371a61cb9b0f9c2e69c3163643/config.md#linux-and-solaris-user